source/version.js

```diff
@@ -18,10 +18,18 @@ exports.getNewVersion = (oldVersion, input) => {
 	return exports.SEMVER_INCREMENTS.includes(input) ? semver.inc(oldVersion, input) : input;
 };
 
-exports.isVersionGreater = (oldVersion, newVersion) => {
-	if (!isValidVersion(newVersion)) {
+const validateVersion = version => {
+	if (!isValidVersion(version)) {
 		throw new Error('Version should be a valid semver version.');
 	}
+};
+
+exports.isVersionGreaterThan = (firstVersion, secondVersion) => {
+	validateVersion(secondVersion);
```

Comment:
Should we validate both given versions or just `secondVersion`?
Previously, only the second version was being validated (probably because the first version was always a constant, valid version).
To me, it seems preferable to just test both versions.

Reply:
Both